src/test/java/marquez/db/mappers/DatasetRowMapperTest.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package marquez.db.mappers;
+
+import static marquez.common.models.CommonModelGenerator.newDatasetName;
+import static marquez.common.models.CommonModelGenerator.newDatasetUrn;
+import static marquez.common.models.CommonModelGenerator.newDescription;
+import static marquez.common.models.Description.NO_DESCRIPTION;
+import static marquez.db.models.DbModelGenerator.newRowUuid;
+import static marquez.db.models.DbModelGenerator.newTimestamp;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatNullPointerException;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.UUID;
+import marquez.UnitTests;
+import marquez.common.models.DatasetName;
+import marquez.common.models.DatasetUrn;
+import marquez.common.models.Description;
+import marquez.db.Columns;
+import marquez.db.models.DatasetRow;
+import org.jdbi.v3.core.statement.StatementContext;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+@Category(UnitTests.class)
+public class DatasetRowMapperTest {
+  private static final UUID ROW_UUID = newRowUuid();
+  private static final Instant CREATED_AT = newTimestamp();
+  private static final Instant UPDATED_AT = CREATED_AT;
+  private static final UUID NAMESPACE_ROW_UUID = newRowUuid();
+  private static final UUID DATASOURCE_ROW_UUID = newRowUuid();
+  private static final DatasetName NAME = newDatasetName();
+  private static final DatasetUrn URN = newDatasetUrn();
+  private static final Description DESCRIPTION = newDescription();
+  private static final UUID CURRENT_VERSION_UUID = newRowUuid();
+
+  @Mock private Object exists;
+  @Mock private ResultSet results;
+  @Mock private StatementContext context;
+
+  @Before
+  public void setup() {
+    MockitoAnnotations.initMocks(this);
```

Comment:
Cool! Really like the reduced boilerplate code on mock creation 💯

Mind opening an issue to update all tests with mock usage to follow this approach? Thanks!

Reply:
thank you! sure, will open an issue shortly